benchmark/util.jl

```diff
@@ -33,5 +33,6 @@ function add_to_suite!(suite, sim_function; p=(3,4,5), s=100, ft=Float32, backen
     end
 end
 
-git_hash = read(`git rev-parse --short HEAD`, String) |> x -> strip(x, '\n')
+waterlily_dir = get(ENV, "WATERLILY_ROOT", "")
+git_hash = read(`git -C $waterlily_dir rev-parse --short HEAD`, String) |> x -> strip(x, '\n')
```

Comment:
I didn't have the time to try it yet, but my problem is that I didn't even have WaterLily`dev`'ed in my case, I wanted to normally `add` it to an environment and run the benchmarks, that's it, so this would fail for my use case.

Reply:
I see... well thanks for the input. I think I will keep it like this for the moment since it is mostly intended for internal usage anyways (and to be fair it should be placed into its small repo once we transition to the Github organisation account). Thanks!